R/backport_linter.R

```diff
@@ -0,0 +1,83 @@
+#' @describeIn linters that checks for usage of unavailable functions. Not reliable for testing r-devel dependencies.
+#' @export
+backport_linter <- function(r_version = "4.0.3") {
```

Comment:
Should the default `r_version` be the current R version?

If a package author wants to use this linter, then one must specify the `r_version` to be consistent with that appears in the package `DESCRIPTION`?

Reply:
Now I used `getRversion()`, I guess we could make it `guess_r_version()` & check the description first before defaulting to `getRversion()`?